dashboard/fixtures/load_fixtures.sh

```diff
@@ -0,0 +1,2 @@
+clickhouse-client --host $CLICKHOUSE_HOST --database tensorzero --query "INSERT INTO JsonInference FORMAT CSV" < /fixtures/json_inference_examples.csv
+clickhouse-client --host $CLICKHOUSE_HOST --database tensorzero --query "INSERT INTO BooleanMetricFeedback FORMAT CSV" < /fixtures/boolean_metric_feedback_examples.csv
```

Comment:
This should not assume `/fixtures` and instead `./fixtures`. You can move to the relevant directory. Otherwise you cannot run this outside of the container.

Reply:
willfix!